CONFIGDOC.md

```diff
@@ -24,12 +34,12 @@ If the build agent already has your desired MATLAB on the path, then you are not
 | Linux&reg;   | /usr/local/MATLAB/R2019a        |
 | Mac          | /Applications/MATLAB_R2019a.app |
 
-## Specify Build Steps
+### Specify Build Steps
 When you set up the **Build** section of the project configuration window, the plugin provides you with the **Run MATLAB Command** and **Run MATLAB Tests** build steps.
 
 If you use a source code management (SCM) system such as Git&trade;, then your project must include the appropriate SCM configuration to check out the code before it can invoke the plugin. If you do not use any SCM systems to manage your code, then an additional build step is required to ensure that the code is available in the Jenkins workspace before the build starts.

```

Comment:
"then your project **should** include..."

"then an additional build step **may be** required"

Neither of these is a hard requirement to use the plugin.

Reply:
Done.